app/Http/routes.php

```diff
@@ -15,6 +15,15 @@
     return view('welcome');
 });
 
+Route::get('tickets','TicketsController@index');
+Route::get('tickets/create','TicketsController@create');
+Route::get('tickets/{id}','TicketsController@show');
+Route::post('tickets/store','TicketsController@store');
+Route::delete('tickets/{id}','TicketsController@destroy');
+Route::get('tickets/{id}/edit','TicketsController@edit');
+Route::post('tickets/{id}/edited','TicketsController@update');
+
```

Comment:
use Route::resource
a 1 liner and generates all those for you
you can check the generated routes with php artisan route:list and adapt to the ones generated